codec.go

```diff
@@ -35,8 +37,9 @@ func setupCodec() error {
 	section := sections[0]
 	offset := offsets[0]
 	var (
-		min uintptr = uintptr(^uint(0))
-		max uintptr = 0
+		min   uintptr = uintptr(^uint(0))
+		max   uintptr = 0
+		addrs         = make([]int, 0, 4*1024)
```

Comment:
It's just a pre-allocation to reduce allocation when the slice grows, 4096 is chosed to try to cover most usecases, I guess alloc on-demand is ok, since this function is called only once.